### PR TITLE
run latest black over the code base

### PR DESCRIPTION
**Pull request by olliethomas**
we're getting comments from confused contributors because of the black update. This PR is just master, but with the latest version of black run over it.

### CODE REVIEW

**Comment by thomkeh** [comment]
I would have preferred if you had removed the trailing commas instead of letting it be split into multiple lines.

**Comment by thomkeh** [comment]
(That's how I dealt with this in EthicML.)

**Comment by olliethomas** [comment]
hmmm... ok - i can do that

File: tests/pandas_test.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ def assert_type(v: T, t: Type[T]) -> None:
     columns=["max_speed", "shield"],
 )
 fd: pd.DataFrame = pd.DataFrame(
-    [[1.0, 2.0], [4.0, 5.0], [7.0, 8.0]], columns=["max_speed", "shield"],
+    [[1.0, 2.0], [4.0, 5.0], [7.0, 8.0]],
+    columns=["max_speed", "shield"],
```
Comment (thomkeh) [inline tests/pandas_test.py:21]:
```suggestion
    [[1.0, 2.0], [4.0, 5.0], [7.0, 8.0]], columns=["max_speed", "shield"]
```

File: pandas-stubs/core/indexing.pyi [hunk 2]
```diff
@@ -106,13 +107,17 @@ class _LocIndexerSeries(Generic[_DType]):
...
+        value: Union[_DType, _np.ndarray, Series[_DType]],
     ) -> None: ...
     @overload
     def __setitem__(self, idx: str, value: _DType) -> None: ...
     @overload
     def __setitem__(
-        self, idx: List[str], value: Union[_DType, _np.ndarray, Series[_DType]],
+        self,
+        idx: List[str],
+        value: Union[_DType, _np.ndarray, Series[_DType]],
```
Comment (thomkeh) [inline pandas-stubs/core/indexing.pyi:120]:
```suggestion
        self, idx: List[str], value: Union[_DType, _np.ndarray, Series[_DType]]
```

File: pandas-stubs/core/indexing.pyi [hunk 3]
```diff
@@ -106,13 +107,17 @@ class _LocIndexerSeries(Generic[_DType]):
     @overload
     # set item
     def __setitem__(
-        self, idx: _MaskType, value: Union[_DType, _np.ndarray, Series[_DType]],
+        self,
+        idx: _MaskType,
+        value: Union[_DType, _np.ndarray, Series[_DType]],
```
Comment (thomkeh) [inline pandas-stubs/core/indexing.pyi:112]:
```suggestion
        self, idx: _MaskType, value: Union[_DType, _np.ndarray, Series[_DType]]
```

File: pandas-stubs/core/indexing.pyi [hunk 4]
```diff
@@ -61,7 +61,8 @@ class _LocIndexerFrame:
     def __getitem__(self, idx: List[_StrLike]) -> DataFrame: ...
     @overload
     def __getitem__(
-        self, idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]],
+        self,
+        idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]],
```
Comment (thomkeh) [inline pandas-stubs/core/indexing.pyi:65]:
```suggestion
        self, idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]]
```

File: pandas-stubs/core/groupby/generic.pyi [hunk 5]
```diff
@@ -17,7 +17,8 @@ class DataFrameGroupBy(GroupBy):
     def __getattr__(self, name: _str) -> SeriesGroupBy: ...
     def __iter__(self) -> Iterator: ...
     def aggregate(
-        self, func: Union[_FunctionLike, List[_FunctionLike], Dict[_str, _FunctionLike]],
+        self,
+        func: Union[_FunctionLike, List[_FunctionLike], Dict[_str, _FunctionLike]],
```
Comment (thomkeh) [inline pandas-stubs/core/groupby/generic.pyi:21]:
```suggestion
        self, func: Union[_FunctionLike, List[_FunctionLike], Dict[_str, _FunctionLike]]
```

File: pandas-stubs/core/frame.pyi [hunk 6]
```diff
@@ -235,7 +243,11 @@ class DataFrame:
     # rename specifying mapper= and axis=
     @overload
     def rename(
-        self, mapper: _Renamer, *, inplace: Literal[True], axis: _AxisType = ...,
+        self,
+        mapper: _Renamer,
+        *,
+        inplace: Literal[True],
+        axis: _AxisType = ...,
```
Comment (thomkeh) [inline pandas-stubs/core/frame.pyi:250]:
```suggestion
        self, mapper: _Renamer, *, inplace: Literal[True], axis: _AxisType = ...
```

File: pandas-stubs/core/frame.pyi [hunk 7]
```diff
@@ -216,7 +220,11 @@ class DataFrame:
     @overload
     def itertuples(self, name: Literal[None], index: bool = ...) -> Iterator[Tuple[Any, ...]]: ...
     @overload
-    def itertuples(self, name: _str, index: bool = ...,) -> Iterator[NamedTuple]: ...
+    def itertuples(
+        self,
+        name: _str,
+        index: bool = ...,
+    ) -> Iterator[NamedTuple]: ...
```
Comment (thomkeh) [inline pandas-stubs/core/frame.pyi:227]:
```suggestion
    def itertuples(self, name: _str, index: bool = ...) -> Iterator[NamedTuple]: ...
```

File: pandas-stubs/core/frame.pyi [hunk 8]
```diff
@@ -207,7 +207,11 @@ class DataFrame:
     def idxmax(self, axis: _AxisType = ...) -> Series: ...
     def idxmin(self, axis: _AxisType = ...) -> Series: ...
     def insert(
-        self, loc: int, column: _str, value: _ListLike, allow_duplicates: bool = ...,
+        self,
+        loc: int,
+        column: _str,
+        value: _ListLike,
+        allow_duplicates: bool = ...,
```
Comment (thomkeh) [inline pandas-stubs/core/frame.pyi:214]:
```suggestion
        self, loc: int, column: _str, value: _ListLike, allow_duplicates: bool = ...
```

File: numpy-stubs/random.pyi [hunk 9]
```diff
@@ -19,12 +19,22 @@ def binomial(n: _IntLike, p: _FloatLike) -> _Int: ...
...
+    n: _ArrayLike[_Int],
+    p: _FloatLike,
+    size: ndarray[_Int] = ...,
+) -> ndarray[_Int]: ...
 @overload
 def binomial(
-    n: _ArrayLike[_Int], p: _ArrayLike[_Float], size: ndarray[_Int] = ...,
+    n: _ArrayLike[_Int],
+    p: _ArrayLike[_Float],
+    size: ndarray[_Int] = ...,
```
Comment (thomkeh) [inline numpy-stubs/random.pyi:37]:
```suggestion
    n: _ArrayLike[_Int], p: _ArrayLike[_Float], size: ndarray[_Int] = ...
```

File: numpy-stubs/random.pyi [hunk 10]
```diff
@@ -19,12 +19,22 @@ def binomial(n: _IntLike, p: _FloatLike) -> _Int: ...
...
+) -> ndarray[_Int]: ...
 @overload
-def binomial(n: _ArrayLike[_Int], p: _FloatLike, size: ndarray[_Int] = ...,) -> ndarray[_Int]: ...
+def binomial(
+    n: _ArrayLike[_Int],
+    p: _FloatLike,
+    size: ndarray[_Int] = ...,
+) -> ndarray[_Int]: ...
```
Comment (thomkeh) [inline numpy-stubs/random.pyi:32]:
```suggestion
def binomial(n: _ArrayLike[_Int], p: _FloatLike, size: ndarray[_Int] = ...) -> ndarray[_Int]: ...
```

File: numpy-stubs/random.pyi [hunk 11]
```diff
@@ -19,12 +19,22 @@ def binomial(n: _IntLike, p: _FloatLike) -> _Int: ...
 @overload
 def binomial(n: _IntLike, p: _FloatLike, size: _IntLike) -> ndarray[_Int]: ...
 @overload
-def binomial(n: _IntLike, p: _ArrayLike[_Float], size: ndarray[_Int] = ...,) -> ndarray[_Int]: ...
+def binomial(
+    n: _IntLike,
+    p: _ArrayLike[_Float],
+    size: ndarray[_Int] = ...,
+) -> ndarray[_Int]: ...
```
Comment (thomkeh) [inline numpy-stubs/random.pyi:26]:
```suggestion
def binomial(n: _IntLike, p: _ArrayLike[_Float], size: ndarray[_Int] = ...) -> ndarray[_Int]: ...
```

File: matplotlib-stubs/pyplot.pyi.in [hunk 12]
```diff
@@ -322,10 +322,16 @@ def fill_between(
...
+    xmin: float = ...,
+    xmax: float = ...,
+    ___line2d_kwargs___,
 ) -> Line2D: ...
 def axvline(
-    x: float = ..., ymin: float = ..., ymax: float = ..., ___line2d_kwargs___,
+    x: float = ...,
+    ymin: float = ...,
+    ymax: float = ...,
+    ___line2d_kwargs___,
```
Comment (thomkeh) [inline matplotlib-stubs/pyplot.pyi.in:334]:
```suggestion
    x: float = ..., ymin: float = ..., ymax: float = ..., ___line2d_kwargs___
```

File: matplotlib-stubs/pyplot.pyi.in [hunk 13]
```diff
@@ -322,10 +322,16 @@ def fill_between(
     **kwargs: Any,
 ) -> PolyCollection: ...
 def axhline(
-    y: float = ..., xmin: float = ..., xmax: float = ..., ___line2d_kwargs___,
+    y: float = ...,
+    xmin: float = ...,
+    xmax: float = ...,
+    ___line2d_kwargs___,
```
Comment (thomkeh) [inline matplotlib-stubs/pyplot.pyi.in:328]:
```suggestion
    y: float = ..., xmin: float = ..., xmax: float = ..., ___line2d_kwargs___
```

**Pull request merged**